projects/laji/src/app/shared-modules/select/select/select.component.ts

```diff
@@ -103,7 +102,7 @@ export class SelectComponent<T extends idType|SelectOption = string> implements
         this.selected = [...this.selected, id] as T[];
       }
     } else {
-      this.selected = isBasic ? [option] as T[] : [id] as T[];
+      this.selected = isBasic ? [option.id] as T[] : [id] as T[];
```

Comment:
It seems like there has been a bug in the original row, but now it would return same kind of result on both cases. Most likely the correct form should be `this.selected = isBasic ? [id] as T[] : [option] as T[];` (Idea is to make this.selected to match the type that was given in the @Input)

Reply:
This wasn't update with dev version, so this issue still was there